pkg/retry/retry_test.go

```diff
@@ -55,6 +56,7 @@ func TestHandleRetry(t *testing.T) {
 				listDeliveries: &listDeliveriesRes{
 					deliveries: []*github.HookDelivery{
 						{
+							ID:         toPtr[int64](101),
```

Comment:
I actually don't know which is more idiomatic since generics are relatively new, but you should also be able to:

```go
toPtr(int64(101))
```